example-shelley/impl/src/Cardano/Ledger/Example.hs

```diff
@@ -0,0 +1,210 @@
+import Shelley.Spec.Ledger.Coin (Coin)
+import Shelley.Spec.Ledger.EpochBoundary (BlocksMade (..), emptySnapShots)
+import Shelley.Spec.Ledger.Metadata (Metadata (Metadata), validMetadatum)
+import Shelley.Spec.Ledger.Scripts (MultiSig)
+import Shelley.Spec.Ledger.PParams as Shelley
+import Shelley.Spec.Ledger.Tx
+  ( ValidateScript (..),
+    validateNativeMultiSigScript,
+    Tx,
+  )
+import Shelley.Spec.Ledger.API
+  ( AccountState (AccountState),
+    ApplyBlock,
+    ApplyTx,
+    CanStartFromGenesis(..),
+    Coin (Coin),
+    DPState (DPState),
+    DState (_genDelegs),
+    EpochState (EpochState),
+    GenDelegs (GenDelegs),
+    GetLedgerView,
+    LedgerState (LedgerState),
+    NewEpochState (NewEpochState),
+    PraosCrypto,
+    PoolDistr (PoolDistr),
+    ShelleyBasedEra,
+    ShelleyGenesis (sgGenDelegs, sgMaxLovelaceSupply, sgProtocolParams),
+    StrictMaybe (SNothing),
+    TxOut (..),
+    UTxOState (UTxOState),
+    balance,
+    genesisUTxO,
+    word64ToCoin,
+  )
+
+import Shelley.Spec.Ledger.TxBody (TxBody (..))
+
+import Shelley.Spec.Ledger.STS.Bbody (BBODY)
+import Shelley.Spec.Ledger.STS.Deleg (DELEG)
+import Shelley.Spec.Ledger.STS.Delegs (DELEGS)
+import Shelley.Spec.Ledger.STS.Delpl (DELPL)
+import Shelley.Spec.Ledger.STS.Epoch (EPOCH)
+import Shelley.Spec.Ledger.STS.Ledger (LEDGER)
+import Shelley.Spec.Ledger.STS.Ledgers (LEDGERS)
+import Shelley.Spec.Ledger.STS.Mir (MIR)
+import Shelley.Spec.Ledger.STS.NewEpoch (NEWEPOCH)
+import Shelley.Spec.Ledger.STS.Newpp (NEWPP)
+import Shelley.Spec.Ledger.STS.Ocert (OCERT)
+import Shelley.Spec.Ledger.STS.Overlay (OVERLAY)
+import Shelley.Spec.Ledger.STS.Pool (POOL)
+import Shelley.Spec.Ledger.STS.PoolReap (POOLREAP)
+import Shelley.Spec.Ledger.STS.Ppup (PPUP)
+import Shelley.Spec.Ledger.STS.Rupd (RUPD)
+import Shelley.Spec.Ledger.STS.Snap (SNAP)
+import Shelley.Spec.Ledger.STS.Tickn (TICKN)
+import Shelley.Spec.Ledger.STS.Tick (TICK, TICKF)
+import Shelley.Spec.Ledger.STS.Upec (UPEC)
+import Shelley.Spec.Ledger.STS.Utxo (UTXO)
+import Shelley.Spec.Ledger.STS.Utxow (UTXOW)
+
+data ExampleEra c
+
+instance CryptoClass.Crypto c => Era (ExampleEra c) where
+  type Crypto (ExampleEra c) = c
+
+instance CryptoClass.Crypto c => UsesValue (ExampleEra c)
+
+instance CryptoClass.Crypto c => UsesTxOut (ExampleEra c) where
+  makeTxOut _ a v = TxOut a v
+
+--------------------------------------------------------------------------------
+-- Core instances
+--------------------------------------------------------------------------------
+
+type instance Core.Value (ExampleEra _c) = Coin
+
+type instance Core.TxBody (ExampleEra c) = TxBody (ExampleEra c)
+
+type instance Core.TxOut (ExampleEra c) = TxOut (ExampleEra c)
+
+type instance Core.Script (ExampleEra c) = MultiSig c
+
+type instance Core.AuxiliaryData (ExampleEra c) = Metadata (ExampleEra c)
+
+type instance Core.PParams (ExampleEra c) = Shelley.PParams (ExampleEra c)
+
+type instance Core.Tx (ExampleEra c) = Tx (ExampleEra c)
```

Comment:
This has now changed, as per https://github.com/input-output-hk/cardano-ledger-specs/pull/2225